.github/ISSUE_TEMPLATE/platform-user-story.md

```diff
@@ -0,0 +1,45 @@
+<!-- 
+What is a User Story?
+A user story is a requirement for any functionality or feature, written down in one or two lines, about one and only one functionality or feature.
+
+What is Acceptance Criteria?
```

Comment:
Can we move this part of the note down to the acceptance criteria section please?